modules/ROOT/pages/hardware-and-software-requirements.adoc

```diff
@@ -41,7 +41,7 @@ The Mule runtime engine passed functional testing against the following software
 | JDK | JDK 1.8.0, JDK 11
 |===
 
-This version of Mule runtime engine is bundled with the Runtime Manager agent plugin version 2.1.9.
+This version of Mule runtime engine is bundled with the Runtime Manager agent plugin version 2.4.5. For Runtime Manager Agent compatibility, see xref:release-notes::runtime-manager-agent/runtime-manager-agent-release-notes.adoc[Runtime Manager Agent Release Notes].
```

Comment:
version came from info at bottom of https://docs.mulesoft.com/release-notes/mule-runtime/mule-4.3.0-release-notes